pages/fundamentals/introduction/index.zh-hans.md

```diff
@@ -3,15 +3,17 @@
 # In this first section, do not translate the words before a colon. For example, do not translate "title:". Do translate the text after "title:"
 title: Web无障碍简介 
 lang: zh-hans
-last_updated: 2019-06-05
+last_updated: 2024-03-07
+first_published: "February 2005"
```

Comment:
@JinMokai I have just added this line, recently introduced in the original version. Would you please translate it? Thanks!